server/src/main/java/keywhiz/service/daos/AclDAO.java

```diff
@@ -529,6 +529,12 @@ protected void evictClient(Configuration configuration, long clientId, long grou
         .execute();
   }
 
+  public ImmutableSet<SecretSeries> getSecretSeriesFor(Group group) {
+    checkNotNull(group);
+
+    return dslContext.transactionResult(configuration -> getSecretSeriesFor(configuration, group));
+  }
```

Comment:
The API for fetching a group's info needs to get a list of the names of all the secrets which that group has access to. The names live on the series and not the contents, however, there was no API exposed to fetch just the series themselves.

Reply:
This all looks good, and I'd hope the Keywhiz maintainers would ask for at least a smoke test added for any new publicly exposed method.

Reply:
It's covered by the existing test suite for the Group resource, which populates and then queries (via this API) an in-memory database.

Reply:
TIL, thanks.